broker/lib/middleware.js

```diff
@@ -105,7 +105,7 @@ exports.checkQuota = function () {
   function shouldCheckQuotaForPlatform(platform, origin) {
     return (platform === CONST.PLATFORM.CF ||
       (platform === CONST.PLATFORM.SM &&
-        origin === CONST.PLATFORM.CF));
+        (origin === CONST.PLATFORM.CF || origin === CONST.PLATFORM.K8S)));
```

Comment:
This condition basically means everything, right ? Should we remove this check altogether ?

Reply:
`platform === CONST.PLATFORM.K8S` is not supported. Or shall we support it?

Reply:
We do not have a usecase for it as of now...but if we have, why not! One can anyways turn off quota check with an additional flag. right ?

Current code would technically work even for `platform === CONST.PLATFORM.K8S` provided the quota APIs support it.

Reply:
Makes sense, I will remove the condition.

Reply:
Done